packages/wearable_rotary/lib/wearable_rotary.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/services.dart';
+
+const String _ROTARY_CHANNEL_NAME = 'flutter.wearable_rotary.channel';
+
+const EventChannel _channel = EventChannel(_ROTARY_CHANNEL_NAME);
+
+Stream<RotaryEvent> get rotaryEvent {
+  _rotaryEvent ??= _channel
+      .receiveBroadcastStream()
+      .map((dynamic event) => _parseEvent(event));
+  return _rotaryEvent!;
+}
+
+Stream<RotaryEvent>? _rotaryEvent;
+
+enum RotaryEvent {
+  CLOCKWISE,
+  COUNTER_CLOCKWISE,
```

Comment:
```suggestion
  clockwise,
  counterClockwise,
```
Note: You need to update REAME as well after this change.